docs/contributing/index.md

```diff
@@ -75,12 +99,12 @@ the stubs appear under `stubs/cmr`.
 ## Pull Request process
 
 1. Ensure you include test coverage for all changes
-1. Ensure your code is formatted properly following this document
-1. Update the documentation and the `README.md` with details of changes to the
+2. Ensure your code is formatted properly following this document
```

Comment:
Use of `1.` for every numbered item is so that we don't have to worry about numbering, particularly when we add, remove, or reorder items. The markdown renderer automatically applies sequential numbering for us.